### PR TITLE
5.x - FIO-8426: changes required for eSignature

### PR DESCRIPTION
**Pull request by TanyaGashtold**
## Link to Jira Ticket

https://formio.atlassian.net/browse/FIO-8426

## Description

**What changed?**

separated some code as new methods

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings
- [ ] My changes include tests that prove my fix is effective (or that my feature works as intended)
- [x] New and existing unit/integration tests pass locally with my changes
- [x] Any dependent changes have corresponding PRs that are listed above

### CODE REVIEW

**Comment by johnformio** [comment]
LGTM.  Would like to get @brendanbond to take a look as well.

File: src/Webform.js [hunk 1]
```diff
@@ -811,6 +811,18 @@ export default class Webform extends NestedDataComponent {
     this.setSubmission(submission);
   }
 
+    /**
+   * @param submission
+   * @param flags
+   * @return {void}
```
Comment (brendanbond) [inline src/Webform.js:817]: this is a HUGE nit, but the incoming PR from Travis will cause the linter to complain about this - can we add descriptions for each?

File: src/components/_classes/component/Component.js [hunk 2]
```diff
@@ -1227,22 +1231,27 @@ export default class Component extends Element {
...
-
-        this.tooltips[index] = tippy(tooltip, {
-          allowHTML: true,
-          trigger: 'mouseenter click focus',
-          placement: 'right',
-          zIndex: 10000,
-          interactive: true,
-          content: this.t(this.sanitize(tooltipText), { _userInput: true }),
-        });
+        this.tooltips[index] = this.createTooltip(tooltip);
```
Comment (brendanbond) [inline src/components/_classes/component/Component.js:1254]: very nice

File: src/Webform.js (hunk 1 above)
Comment (TanyaGashtold) [reply src/Webform.js:818]: Done!

**Review by brendanbond — APPROVED** [review]

**Pull request merged**